test/vni-manager-mocha.js

```diff
@@ -0,0 +1,377 @@
+/**
+ * File: vni-manager-mocha.js
+ * Unit tests for the vni APIs defined in src/vni-manager.js
+ */
+
+var chai = require('chai');
+var expect = chai.expect;
+var should = chai.should();
+
+var componentFactory = require('../src/noflo-component-factory');
+var createState = require('../src/create-state');
+var inputStates = require('../src/input-states');
+var promiseOutput = require('../src/promise-output');
+
+var test = require('./common-test');
+
+var vniManager = require('../src/vni-manager');
+
+describe("vni-manager", function() {
+
+    beforeEach(function() {
+
+        node = test.createComponent(componentFactory({
```

Comment:
Should we rename createComponent to createNode?  I'm getting confused about which objects are components (i.e., node factories) and which are nodes (i.e., node instances).